### PR TITLE
Docs: Fix links of GLTFLoader page

### PR DESCRIPTION
**Pull request by Mugen87**
Minor fix similar to #14196

### CODE REVIEW

**Comment by Mugen87** [comment]
https://rawgit.com/Mugen87/three.js/dev14/docs/#examples/loaders/GLTFLoader

File: docs/examples/loaders/GLTFLoader.html [hunk 1]
```diff
@@ -60,7 +60,7 @@ <h2>Example</h2>
 				gltf.asset; // Object
 
 			},
-			// called while loading is progressing
+			// called when loading is in progresses
```
Comment (donmccurdy) [inline docs/examples/loaders/GLTFLoader.html]: ^may need to rebase.

Comment (Mugen87) [reply docs/examples/loaders/GLTFLoader.html]: Should be okay now :+1:

**Review by donmccurdy — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**